vaadin-date-time-picker-flow/src/main/java/com/vaadin/flow/component/datetimepicker/DateTimePicker.java

```diff
@@ -153,6 +199,28 @@ public DateTimePicker(LocalDateTime initialDateTime,
         addValueChangeListener(listener);
     }
 
+    /**
+     * Convenience constructor to create a date time picker with a pre-selected
+     * date and time in current UI locale format, a {@link ValueChangeListener}
+     * and a label.
+     *
+     * @param label
+     *            the label describing the date picker
```

Comment:
"date time picker" instead of "date picker"

Reply:
Updated.